docs/contributing/03-stories/story-16.md

```diff
@@ -0,0 +1,100 @@
+# User Story 16 - Wing for TypeScript
+
+> **Status**: Draft
+
+Anders loves TypeScript, but he also loves the amazing experience Wing offers for building cloud applications.
+
+It's Andres' lucky day because Wing Cloud had just announced that very soon TypeScript developers are able to start leveraging Wing libraries from the comfort of
+their favorite programming language.
+
+He got an invite for a pre-release. The invite explains that initially, there is only support for `cloud.Bucket` and `cloud.Function` and for consuming [winglibs](https://github.com/winglang/winglibs).
+
+The example looks like this:
+
+First, install some libraries:
+
+```sh
+$ npm i winglang wingsdk @winglibs/cloud @winglibs/checks
+```
+
+```ts
+// main.ts
+import { inflight, App } from "wingsdk";
```

Comment:
Assuming this is the same wingsdk that's used elsewhere, which version of the SDK takes priority if the node_modules version and the CLI version conflict?

This problem technically already exists but will be more prevalent when users are commonly installing the SDK themselves. I believe currently the CLI version takes precedence, but I'm guess that would have to change. WDYT?

Reply:
As mentioned above, this is *not* `@winglang/sdk`, it's a new library designed for TypeScript users.

I think your concern still holds though but as you observed, we already have this issue, so I think we should discuss this outside the scope of the TypeScript experience.